packer/core.go

```diff
@@ -7,8 +7,8 @@ import (
 
 	ttmp "text/template"
 
-	multierror "github.com/hashicorp/go-multierror"
-	version "github.com/hashicorp/go-version"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-version"
```

Comment:
Go imports will re-name a pkg when the pkg path ex: `go-version` doesn't match the exported name ex: `version`. So I think that #8686 will change this back